queries.sql

```diff
@@ -1,10 +1,41 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE NAME LIKE '%mon';
-SELECT NAME FROM animals WHERE DATE_OF_BIRTH >= '2016-01-01' and DATE_OF_BIRTH <= '2019-12-31';
-SELECT NAME FROM animals WHERE NEUTERED = true AND SCAPE_ATTEMPTS < 3;
-SELECT DATE_OF_BIRTH FROM animals WHERE NAME IN ('Agumon',' Pikachu');
-SELECT NAME, SCAPE_ATTEMPTS FROM animals WHERE WEIGHT_KG > 10.5;
-SELECT * FROM animals WHERE NEUTERED = true;
-SELECT * FROM animals WHERE NAME != 'GABUMON';
-SELECT * FROM animals WHERE WEIGHT_KG >= 10.4 AND WEIGHT_KG <= 17.3;
\ No newline at end of file
+--What animals belong to Melody Pond?
+SELECT a.*
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Melody Pond';
+
+--List of all animals that are pokemon (their type is Pokemon).
+select a.* FROM animals a join species s ON a.species_id = s.id where s.name = 'Pokemon';
+
```

Comment:
- [ ] As required for the project, kindly ensure that you add a screenshot that lists all animals that are `pokemon` as right now you repeat the same screenshot that lists animals belong to Melody Pond instead of those that are pokemon (their type is Pokemon). This ensure that you run the queries with success and get the correct result.

Reply:
done!
this is the table species:
<img width="403" alt="image" src="https://github.com/benja27/vet_clinic/assets/50673336/11a98ff7-18bb-41b2-9c2e-8107b7be13fa">

and this is the result of the query:
List of all animals that are pokemon (their type is Pokemon).
![image](https://github.com/benja27/vet_clinic/assets/50673336/e70debfc-9368-4eeb-9b12-32bbfbacab73)